### DISCUSSION TITLE
Specify your own build sample program

### DISCUSSION THREAD

**Discussion by phaze101** [prb28/vscode-amiga-assembly#242] [Ideas]
Is it possible to add an option to create a project from your own sample project?

Reason for this is I want to provide to my followers different sample startup projects also I think it is handy to have.

**Comment by prb28** [comment]
You can change in the settings the "Example project branches URL" ("https://github.com/prb28/vscode-amiga-wks-example/archive/refs/heads") set it to you github project.

**Reply by phaze101** [reply]
@prb28 so what are the tags. Since this is what stopped me from trying and why I suggested it.

**Reply by prb28** [reply]
If your project has tags it will take the corresponding tag to the extension version. Otherwise it will take the master branch.
